var/spack/repos/builtin/packages/nf-core-tools/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class NfCoreTools(PythonPackage):
```

Comment:
Is this primarily used as a command line tool or is it primarily imported in a Python script? Any reason to change from the default name `py-nf-core`?

Reply:
hi @adamjstewart , it is one of those cases where the tool "happens" to be written in Python, but is used entirely as a command line tool

Reply:
Should we instead call it `nf-core` then?

Reply:
I recommend `nf-core-tools`, because the `nf-core` projects has many repos/sub-projects, with `tools` being the CLI interface